code/Gamemodes/Modes/FreeForAllGamemode.cs

```diff
@@ -68,6 +68,14 @@ internal override void Start()
 		var firstGrub = PlayerGrubOrder[firstPlayer].Dequeue();
 		PlayerGrubOrder[firstPlayer].Enqueue( firstGrub );
 		ActivePlayerId = firstPlayer.Id;
+
+		// Landmine Spawning
+		for ( var i = 0; i < GrubsConfig.LandmineSpawnCount; i++ )
+		{
+			var spawnPos = GrubsTerrain.Instance.FindSpawnLocation( inAir: false );
+			LandmineUtility.Instance.Spawn( spawnPos );
```

Comment:
Are we ensuring the landmines aren't spawning on grubs?

Reply:
Discussed - Grubs can spawn on landmines but they will not trigger until they move